Lib/asyncio/base_events.py

```diff
@@ -912,8 +916,18 @@ def create_datagram_endpoint(self, protocol_factory,
 
             exceptions = []
 
-            if reuse_address is None:
-                reuse_address = os.name == 'posix' and sys.platform != 'cygwin'
+            # bpo-37228
+            if reuse_address is not _unset:
+                if reuse_address:
+                    raise ValueError("Passing `reuse_address=True` is no "
+                                     "longer supported, as the usage of "
+                                     "SO_REUSEPORT in UDP poses a significant "
+                                     "security concern.")
+                else:
+                    warnings.warn("The *reuse_address* parameter has been "
+                                  "deprecated as of 3.5.10 and is scheduled "
```

Comment:
Can you make that 3.6.10 for this branch, please?

Reply:
Yeah, good catch.

Reply:
I'll do the same with the doc version as well, I was focused on fixing the merge conflicts and forgot to change the versions.

Reply:
Done.